docs/operator-manual/user-management/keycloak.md

```diff
@@ -19,62 +28,97 @@ Enable the __Client authentication__.
 
 Configure the client by setting the __Root URL__, __Web origins__, __Admin URL__ to the hostname (https://{hostname}).
 
-Also you can set __Home URL__ to your _/applications_ path and __Valid Post logout redirect URIs__ to "+".
+Also you can set __Home URL__ to _/applications_ path and __Valid Post logout redirect URIs__ to "https://{hostname}/applications".
 
 The Valid Redirect URIs should be set to https://{hostname}/auth/callback (you can also set the less secure https://{hostname}/* for testing/development purposes,
 but it's not recommended in production).
 
 ![Keycloak configure client](../../assets/keycloak-configure-client.png "Keycloak configure client")
 
-Make sure to click __Save__. There should be a tab called __Credentials__. You can copy the Secret that we'll use in our ArgoCD 
-configuration.
+Make sure to click __Save__.
+
+There should be a tab called __Credentials__. You can copy the Client Secret that we'll use in our ArgoCD configuration.
 
 ![Keycloak client secret](../../assets/keycloak-client-secret.png "Keycloak client secret")
 
-## Configuring the groups claim
+### Configuring ArgoCD OIDC
 
-In order for ArgoCD to provide the groups the user is in we need to configure a groups claim that can be included in the authentication token.
-To do this we'll start by creating a new __Client Scope__ called _groups_.
+Let's start by storing the client secret you generated earlier in the argocd secret _argocd-secret_.
 
-![Keycloak add scope](../../assets/keycloak-add-scope.png "Keycloak add scope")
+You can patch it with value copied previously:
+```bash
+kubectl -n argo-cd patch secret argocd-secret --patch='{"stringData": { "oidc.keycloak.clientSecret": "83083958-8ec6-47b0-a411-a8c55381fbd2" }}'
```

Comment:
nit: Instead of passing the random string value, we can maybe set it to `<REPLACE_WITH_CLIENT_SECRET>`. WDYT?

Reply:
I think you are absolutely right, I have updated my pull request :)